### PR TITLE
fix master jest issue

### PR DESCRIPTION
**Pull request by kalidiagne**
# Description

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Review by searchableguy — COMMENTED** [review]
Remote the jest plugin if you are replacing it with vitest?

**Comment by kalidiagne** [comment]
> Remote the jest plugin if you are replacing it with vitest?

actually we removed already jest everywhere https://github.com/unlock-protocol/unlock/issues/10476 (except unlock-js)

**Review by julien51 — APPROVED** [review]
perfect!

File: packages/eslint-config/src/base.js [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:json/recommended',
+  ],
   env: {
     es6: true,
     node: true,
     browser: true,
-    jest: true,
+    jest: false,
```
Comment (julien51) [inline packages/eslint-config/src/base.js:11]:
```suggestion
```

**Review by julien51 — APPROVED** [review]

**Pull request merged**